### PR TITLE
Export utils

### PR DESCRIPTION
**Pull request by dimityrmirchev**
**What this PR does / why we need it**:
This PR exposes some of the utility functions so that they can be used in rule implementations outside of this repository.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```feature user

```

### CODE REVIEW

File: pkg/kubernetes/utils/utils.go [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+
+		if len(nodeList.Continue) == 0 {
+			return nodes, nil
+		}
+	}
+}
+
+// GetWorkers return all workers for a given namespace, or all namespaces if it's set to empty string "".
+// It retrieves workers by portions set by limit.
+func GetWorkers(ctx context.Context, c client.Client, namespace string, limit int64) ([]extensionsv1alpha1.Worker, error) {
```
Comment (AleksandarSavchev) [inline pkg/kubernetes/utils/utils.go:90]: Is `pkg/kubernetes/utils` the right place for `GetWorkers()` considering that the `Worker` resource is Gardener specific? I think it should stay in `internal`. WDYT?

**Review by AleksandarSavchev — COMMENTED** [review]
Nice PR! Only 1 minor suggestion.

File: pkg/kubernetes/utils/utils.go (hunk 1 above)
Comment (dimityrmirchev) [reply pkg/kubernetes/utils/utils.go:90]: Yes, good point. I will find a better place for this function.

**Review by AleksandarSavchev — APPROVED** [review]
/lgtm

**Pull request merged**